runtimes/common/src/benchmarks.rs

```diff
@@ -27,3 +34,67 @@ impl<A> From<(A, Vec<u8>)> for DummySignature {
 		DummySignature
 	}
 }
+
+pub mod treasury {
+	use super::*;
+
+	pub struct BenchmarkHelper<T>(PhantomData<T>);
+
+	impl<T> ArgumentsFactory<(), AccountIdOf<T>> for BenchmarkHelper<T>
+	where
+		T: pallet_balances::Config + frame_system::Config,
+		<T as pallet_balances::Config>::Balance: From<u128>,
+		<T as frame_system::Config>::AccountId: From<sp_runtime::AccountId32>,
+	{
+		fn create_asset_kind(_seed: u32) {}
+
+		fn create_beneficiary(seed: [u8; 32]) -> AccountIdOf<T> {
+			let who: AccountIdOf<T> = AccountId::from(seed).into();
+
+			// endow account with some funds
+			let result =
+				<pallet_balances::Pallet<T> as frame_support::traits::fungible::Mutate<AccountIdOf<T>>>::mint_into(
+					&who,
+					KILT.into(),
+				);
+
+			debug_assert!(
+				result.is_ok(),
+				"Could not create account for benchmarking treasury pallet"
+			);
+
+			who
+		}
+	}
+}
+
+pub mod xcm_benchmarking {
```

Comment:
We're already in the `benchmark` module, so perhaps you can call this `xcm`?

Reply:
Good point: [f7e5129](https://github.com/KILTprotocol/kilt-node/pull/655/commits/f7e51296002107e04d22f4317f75d4b3a6822194)